test/fixtures/games_20.csv

```diff
@@ -5,7 +5,7 @@ game_id,season,type,date_time,away_team_id,home_team_id,away_goals,home_goals,ve
 2016030173,20162017,Postseason,4/18/17,24,20,3,2,BMO Field,/api/v1/venues/null
 2014030412,20142015,Postseason,6/6/15,16,14,3,2,Audi Field,/api/v1/venues/null
 2014030413,20142015,Postseason,6/9/15,14,16,3,2,Gillette Stadium,/api/v1/venues/null
-2014030131,20142015,Postseason,4/16/15,5,3,1,2,BBVA Stadium,/api/v1/venues/null
+2014030131,20142015x,Postseason,4/16/15,5,3,1,2,BBVA Stadium,/api/v1/venues/null
```

Comment:
late night attempt to prove sanity. it failed

Reply:
![](https://media3.giphy.com/media/NPyHgTkMStCXC/giphy.gif?cid=5a38a5a2c5fedcd0f28f844ac75c978695c91bd4bc07bb74&rid=giphy.gif)

Reply:
Gif game is out of control